[2022-02-09 09:01:30,822] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-12-01T23:00:00+00:00 [queued]>
[2022-02-09 09:01:30,839] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-12-01T23:00:00+00:00 [queued]>
[2022-02-09 09:01:30,839] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:01:30,840] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:01:30,840] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:01:30,860] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2021-12-01 23:00:00+00:00
[2022-02-09 09:01:30,869] {standard_task_runner.py:52} INFO - Started process 595 to run task
[2022-02-09 09:01:30,881] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'local_to_gcs_task', 'scheduled__2021-12-01T23:00:00+00:00', '--job-id', '523', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingestee_gcs_dag.py', '--cfg-path', '/tmp/tmp1ok75y65', '--error-file', '/tmp/tmp903pt_qz']
[2022-02-09 09:01:30,883] {standard_task_runner.py:77} INFO - Job 523: Subtask local_to_gcs_task
[2022-02-09 09:01:30,997] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-12-01T23:00:00+00:00 [running]> on host cc021fb126ed
[2022-02-09 09:01:31,073] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:01:31,114] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-01T23:00:00+00:00
[2022-02-09 09:01:31,676] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 09:01:31,691] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=local_to_gcs_task, execution_date=20211201T230000, start_date=20220209T090130, end_date=20220209T090131
[2022-02-09 09:01:31,739] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:01:31,789] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-09 09:17:21,101] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-12-01T23:00:00+00:00 [queued]>
[2022-02-09 09:17:21,120] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-12-01T23:00:00+00:00 [queued]>
[2022-02-09 09:17:21,121] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:17:21,121] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:17:21,121] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:17:21,136] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2021-12-01 23:00:00+00:00
[2022-02-09 09:17:21,144] {standard_task_runner.py:52} INFO - Started process 767 to run task
[2022-02-09 09:17:21,156] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'local_to_gcs_task', 'scheduled__2021-12-01T23:00:00+00:00', '--job-id', '629', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingesteee_gcs_dag.py', '--cfg-path', '/tmp/tmp3h5hy3hu', '--error-file', '/tmp/tmpp8t804x1']
[2022-02-09 09:17:21,157] {standard_task_runner.py:77} INFO - Job 629: Subtask local_to_gcs_task
[2022-02-09 09:17:21,254] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-12-01T23:00:00+00:00 [running]> on host 75953ff4f3d5
[2022-02-09 09:17:21,306] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:17:21,328] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-01T23:00:00+00:00
[2022-02-09 09:17:21,854] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 09:17:21,875] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=local_to_gcs_task, execution_date=20211201T230000, start_date=20220209T091721, end_date=20220209T091721
[2022-02-09 09:17:21,935] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:17:21,984] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
